### ISSUE TITLE
Support 7.5 methods

### ISSUE THREAD

**Issue by weirdan** [psalm/phpunit-psalm-plugin#5]
https://github.com/sebastianbergmann/phpunit/blob/master/ChangeLog-7.5.md

Psalm could provide type assertions for `assertIs*` methods.

This raises the question of how to support version-dependent stubs.

**Comment by SignpostMarv** [comment]
sub-packages ?
conditionals based on composer.lock ?
aliasing ?

I would imagine aliasing would be cheapest but most complex long-term; i.e. `TestCase::assertIsString(mixed $maybe)` maps directly to `TestCase::assertInternalType('string', mixed $maybe)`

**Comment by weirdan** [comment]
`composer.lock` checks are probably easiest to implement. Something along the lines of:
```php
$phpunitVersion = \Muglug\PackageVersions\Versions::getShortVersion('phpunit/phpunit');
if (version_compare($phpunitVersion, '7.5.0', '>=')) {
   $psalm->addStubFile(__DIR__ . '/' . 'stubs/PhpUnit75.php');
}
```

If psalm is installed globally it will fetch global phpunit version.

**Comment by weirdan** [comment]
To provide more concrete example: https://github.com/weirdan/doctrine-psalm-plugin/blob/b5d7d1467ec61a1c6b72669dbb304f161e910326/tests/_support/Helper/Psalm.php#L72-L82

```php
use Composer\Semver\Comparator;
use Composer\Semver\VersionParser;
use Muglug\PackageVersions\Versions;

        $currentVersion = Versions::getShortVersion('vimeo/psalm');

        $parser = new VersionParser;

        $currentVersion =  $parser->normalize($currentVersion);
        $version = $parser->normalize($version);

        $result = Comparator::compare($currentVersion, $operator, $version);
```

Same should be possible for any locally installed package.
